src/platform/graphics/texture.js

```diff
@@ -644,6 +661,15 @@ class Texture {
         return this._mipmaps;
     }
 
+    /**
+     * Gets the number of mip levels.
+     *
+     * @type {number}
+     */
+    get mipLevelCount() {
```

Comment:
`numLevels`?

Precedents:

* `IndexBuffer#numIndices`
* `LightCompnent#numCascades`
* `ParticleSystemComponent#numPartices`
* `VertexBuffer#numVertices`
* ...and quite a lot of internal properties following this naming.

Reply:
yeah good point. I so wish JS was a real language, where I can just 'rename' symbol :(
We have many uses of this name and only some need to change.